versions.cmake

```diff
@@ -0,0 +1,45 @@
+superbuild_set_revision(embree
+  URL     "https://www.paraview.org/files/dependencies/embree-v3.13.1.tar.gz"
```

Comment:
We should use github paths

Reply:
This will go in common superbuild anyway